style.css

```diff
@@ -320,3 +320,60 @@ section {
   display: flex;
   flex-direction: column;
 }
+
+.Contact-form {
+  background-color: #6070ff;
+  background-image: url("./images/headerBackgd.png");
+  background-size: cover;
+  border-radius: 60px 0 0 0;
+}
+
+form ul {
+  display: flex;
+  flex-direction: column;
+}
+
+#name,
+#mail,
+#textarea {
+  width: 350px;
+  border-radius: 7px;
+  border: none;
+  margin: 5px;
+  padding-left: 20px;
+}
+
+#textarea {
+  padding-top: 20px;
+}
+
+#name,
+#mail {
+  height: 40px;
+}
+
+form button {
+  width: 150px;
+  height: 50px;
+  border-radius: 7px;
+  border: none;
+}
+
+form button:hover {
+  cursor: pointer;
+  background-color: skyblue;
+}
```

Comment:
- [ ] Awesome job implementing the hover effect, but it's better to stick to the design(color, background), please make sure your button interactions are the same design with Figma's one

![image](https://user-images.githubusercontent.com/59416136/155173619-c09a37ea-9181-4750-aca8-90bb98b990ee.png)